README.md

```diff
@@ -84,6 +84,17 @@ Delete cookie:
 Cookies.remove('name');
 ```
 
+Please notice that removing unexisting cookie does not raise any exception nor return any value
+
+```javascript
+var whatWasTheValue = Cookies.remove('nothing');
+if (whatWasTheValue === undefined) {
+  console.log("I will be excuted");
+} else {
+  console.log("I will never be executed");
```

Comment:
Or maybe:

```
Cookies.remove('name'); // doesn't do anything
```

Or:

```
Cookies.remove('name'); // noop
```

The last one can have accessibility problems because not everyone knows what the `noop` concept means.

The reason why I am suggesting those approaches is because it might be interesting to focus on the least amount of text that fixes any confusion. The README is in a state where any new addition will increase the friction because there will be more content, so it might be worth putting some effort to not add new paragraphs unless essential.

Since you just started with the library you don't have the maintainers bias on the README because we already know how the API works and we can't make it better. Your feedback is really really valuable! Thanks.